src/client/cli/commands/publish.js

```diff
@@ -41,6 +41,12 @@ module.exports = {
       description: 'A jq filter that produces a mediachain identifier from your input object.  ' +
       'Will be applied after `jqFilter`, and does not modify the object itself.\n'
     },
+    prefix: {
+      required: true,
```

Comment:
Perhaps we should make it optional?
It might be the case that the user's objects already have properly prefixed ids, and it's cumbersome to have to strip and re-add